cypress.config.js

```diff
@@ -16,6 +16,19 @@ module.exports = defineConfig({
             password: '12345Qwert!',
           };
         },
+        generateUserDataSettings() {
+          let randomNumber = Math.ceil(Math.random(1000) * 1000);
```

Comment:
Please use 'faker.random.number({ min: 1000, max: 10000 })' to generate a random number between 1000 and 10000.